<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="topLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>584</width>
        <height>301</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="topLayout" stretch="0,1">
       <property name="margin">
        <number>2</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="entries"/>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnAddRecipient">
       <property name="text">
        <string>Add &amp;Recipient</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../icons/add.png</normaloff>../icons/add.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClearAll">
       <property name="text">
        <string>Clear &amp;All</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../icons/remove.png</normaloff>../icons/remove.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnSend">
       <property name="text">
        <string>Send</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../icons/send.png</normaloff>../icons/send.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>scrollArea</tabstop>
  <tabstop>btnAddRecipient</tabstop>
  <tabstop>btnClearAll</tabstop>
  <tabstop>btnSend</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
